source/interfaces/PaginationProperties.ts

```diff
@@ -1,10 +1,31 @@
 export default interface PaginationProperties {
+  count?: string;
+  data?: string;
+  empty?: string;
+  first?: string;
+  headers?: boolean;
+  offsetParameter?: string;
+  last?: string;
+  next?: string;
+  page?: string;
+  pageParameter?: string;
+  pages?: string;
+  sizeParameter?: string;
+  total?: string;
+}
+
+export interface ResolvedPaginationProperties {
```

Comment:
Shouldn't this be in another file? (`source/interfaces/ResolvedPaginationProperties.ts`)

Reply:
Done